src/renderer/components/EnhancedTable/__tests__/AdditionalInfoRows.spec.js

```diff
@@ -1,17 +1,20 @@
 import { render } from 'testingUtility';
 import { AdditionalInfoRows } from '../AdditionalInfoRows';
-import { Cloud } from '../../../../common/__tests__/MockCloud';
+import { Cloud, mkCloudJson } from '../../../../common/Cloud'; // MOCKED
+
+jest.mock('../../../../common/Cloud');
 
 describe('/renderer/components/EnhancedRable/AdditionalInfoRows', () => {
   let fakeCloud;
+  let fakeCloudJson;
 
   beforeEach(() => {
-    fakeCloud = new Cloud('http://foo.com', {
+    fakeCloudJson = mkCloudJson({
```

Comment:
Unless you need the JSON separately (i.e. because you need to initialize the CloudStore with it), you should just create the Cloud directly instead of having the JSON separate. That will keep the test file more concise and no one will wonder why this JSON variable exists. So just replace this with
```suggestion
    fakeCloud = new Cloud({
```

Reply:
Done